### DISCUSSION TITLE
Feature Request: Show all icons in system tray

### DISCUSSION THREAD

**Discussion by christophjodexnus** [valinet/ExplorerPatcher#2502] [Ideas]
Hello Everyone,

i like to see always all icons in the system tray. Can this be added? Then it's a perfect solution for me :)

Regards Christoph

**Comment by pyrates999** [comment]
https://www.makeuseof.com/system-tray-icons-windows-11/

**Comment by Maxximuss** [comment]
You already can, there is an option for that.
